### PR TITLE
Update

### PR DESCRIPTION
**Pull request by mm0**
Update code to render at 6x and loop through test1-3 cams.  Update config to disable devtools.  Code style formatting.

### CODE REVIEW

File: video-encoder-ec2/src/ffmpegWrapper.js [hunk 1]
```diff
@@ -17,16 +17,16 @@ async function ffmpegCreateVideoFromFrames(imageDirectory, fps, resolution) {
...
         return reject();
       } else {
-        if (fps > 30) {
-          console.log('fps too high, it must be less than or equal to 30');
-          return reject();
-        }
+        // if (fps > 30) {
+        //   console.log('fps too high, it must be less than or equal to 30');
+        //   return reject();
+        // }
```
Comment (rimutaka) [inline video-encoder-ec2/src/ffmpegWrapper.js]: Why is this check commented out? Do we check elsewhere?

**Comment by mm0** [comment]
This was in there by the original developer, I ended up using this value to
specify the speed of the image (going from 1x to 6x)

File: video-encoder-ec2/src/index.js [hunk 2]
```diff
@@ -112,8 +112,8 @@ function getConfig(event) {
 async function processVideo(event) {
   return new Promise(async(resolve, reject) => {
     // get config
-    const config = await
-    getConfig(event);
+    const config     = await
```
Comment (rimutaka) [inline video-encoder-ec2/src/index.js]: ??? :)

**Comment by mm0** [comment]
Sorry my IDE is adding a newline after “await” for some reason

**Pull request merged**